frontend/www/js/omegaup/components/common/Navbar.test.ts

```diff
@@ -76,4 +78,49 @@ describe('Navbar.vue', () => {
 
     expect(wrapper.findComponent(UserObjectivesQuestions).exists()).toBe(true);
   });
+
+  it('Should show the information of the next registered contest when the user is registered to a current or upcoming contest', () => {
+    const wrapper = shallowMount(common_Navbar, {
+      propsData: {
+        ...propsData,
+        ...{
+          fromLogin: true,
+          nextRegisteredContest: {
+            admission_mode: '',
+            alias: 'prueba',
+            contest_id: 1,
+            contestants: 10,
+            description: 'Este es un concurso de prueba',
+            duration: null,
+            finish_time: new Date('2023-08-20'),
+            last_updated: new Date('2023-08-16'),
+            organizer: '',
+            original_finish_time: new Date('2023-08-17'),
+            participating: true,
+            problemset_id: 1,
+            recommended: true,
+            rerun_id: 1,
+            score_mode: null,
+            scoreboard_url: null,
+            scoreboard_url_admin: null,
+            start_time: new Date('2023-08-16'),
+            title: '',
+            window_length: null,
+          },
+        },
+      },
+    });
+
+    expect(wrapper.findComponent(user_NextRegisteredContest).exists())
+      .toBeTruthy;
```

Comment:
nit.
```suggestion
      .toBeTruthy();
```
es una función, así que por convención, hay que mandarla llamar correctamente